src/common/Uno.Common/Diagnostics/PlatformDetection.cs

```diff
@@ -17,6 +17,10 @@ public static class PlatformDetection
         public static readonly bool IsArm;
         public static readonly bool Is64Bit;
 
+        public static string UserDirectory => IsWindows
+                ? Environment.GetEnvironmentVariable("USERPROFILE")
```

Comment:
I think this is questionable. IIRC, `$USERPROFILE` isn't always the right environment variable on Windows. In Git for Windows, we try `$HOME` first, then fall back to `$HOMEDRIVE/$HOMEPATH`, then finally fall back to `$USERPROFILE`.

Not consulting `$HOME` would be a regression for people who setup a custom home-dir outside of `C:\Users\`, for instance...

Reply:
Great input, thanks!

I'll change the implementation to be like Git for Windows.

Reply:
Fixed now.